core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -170,6 +170,7 @@ abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with Sas
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
   this.serverConfig.setProperty(KafkaConfig.MinInSyncReplicasProp, "3")
   this.serverConfig.setProperty(KafkaConfig.DefaultReplicationFactorProp, "3")
+  this.serverConfig.setProperty(KafkaConfig.ConnectionsMaxReauthMsProp, "100")
```

Comment:
In each of the tests, we could check metrics (expired-and-killed in some tests and re-authenticated where expected)

Reply:
Was unable to get this working, and it is almost 1am here at this point.  Will look again ASAP.

Reply:
Actually, I just got it working.